test/distributed/org/apache/cassandra/distributed/upgrade/UpgradeTestBase.java

```diff
@@ -90,7 +90,7 @@ public UpgradeableCluster.Builder builder()
     public static final Semver v30 = new Semver("3.0.0-alpha1", SemverType.LOOSE);
     public static final Semver v3X = new Semver("3.11.0", SemverType.LOOSE);
     public static final Semver v40 = new Semver("4.0-alpha1", SemverType.LOOSE);
-    public static final Semver v41 = new Semver("4.1-alpha1", SemverType.LOOSE);
+    public static final Semver v41 = new Semver("4.1", SemverType.LOOSE);
```

Comment:
@Maxwell-Guo why have you changed this?

Reply:
@smiklosovic  Thank you for you review .There was something urgent yesterday at that time , so I seems that I have forgot to check this place when submitting.
I have changed it back.